phoenicis-tests/src/main/java/org/phoenicis/tests/MockedRepository.java

```diff
@@ -38,17 +38,21 @@
 
     private static class MockRepository implements Repository {
         @Override
-        public List<CategoryDTO> fetchInstallableApplications() {
-            return Collections
-                    .singletonList(
-                            new CategoryDTO.Builder()
-                                    .withApplications(Arrays.asList(new ApplicationDTO.Builder().withName("Engines")
-                                            .withScripts(Arrays.asList(new ScriptDTO.Builder().withScriptName("Wine")
-                                                    .withCompatibleOperatingSystems(Arrays.asList(OperatingSystem.LINUX,
-                                                            OperatingSystem.MACOSX))
-                                                    .withScript(wineScript()).build()))
-                                            .build()))
-                                    .withName("Functions").build());
+        public RepositoryDTO fetchInstallableApplications() {
+            return new RepositoryDTO.Builder()
+                    .withCategories(
+                            Collections.singletonList(
+                                    new CategoryDTO.Builder()
+                                            .withApplications(Arrays.asList(new ApplicationDTO.Builder().withName(
```

Comment:
I guess in this particular case, we should build the objects step by step and not in one big call. Anyways this is completely unrelated to the PR.